demos/python_demos/smart_library_demo/smart_library_demo.py

```diff
@@ -0,0 +1,252 @@
+import sys, os
+import argparse
+import numpy as np
+import cv2 as cv
+import json
+
+sys.path.insert(0,'src/modules')
+import face_recognizer as fr 
+import book_recognizer as br
+
+sys.path.insert(0, 'src/infrastructure')
+from DynamicDatabase import *
+
+uknownID = -1
+count = 0
+
+def CreateArgparse():
+    parser = argparse.ArgumentParser(description='Smart Library Sample') 
+    parser.add_argument('-fr', type = str,
+                        dest = 'fRec', required=True, 
```

Comment:
```suggestion
                        required=True,
```
The value of dest is normally inferred from the option strings. The same goes for the rest keys.